### PR TITLE
1063 catalog home page UI

### LINKED ISSUE #1063: Catalog home page UI
UI changes for catalog home page
Change the image and cards.
See zeplin https://app.zeplin.io/project/62b9dba49dd2b211410e608e/screen/62b9dbf580e7c81240864a19 for reference

### PR DESCRIPTION
**Pull request by minump**
## Description
Catalog home page UI changes.
Updated styling of cards and "Show all" "Capabilities" "Talents" buttons. Added new bookshelf image in title.
Removed unwanted styling classes from home.html. Updated css classes in style.css
See http://localhost:5050/catalog/ page for new UI

**Fixes #1063**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [x] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: contributions/catalog/webapps/templates/contribute/home.html [hunk 1]
```diff
@@ -200,13 +129,13 @@ <h2 id="title"> Contributions Catalog/Packager </h2>
...
                 {% for capability in cap_json %}
                     <div class="column capability">
-                        <div class="content">
-                            <a class="card=link" href="{{ url_for('contribute.capability_details', contribution_id=capability['cont_id'], id=capability['id']) }}">
-                            <h4 class="capability_name">{{ capability|filter_nested_dict(["name"]) }}</h4>
-                            <p class="capability_description">{{ capability|filter_nested_dict(["description"]) }}</p>
-                            <p class="contributors">{{ capability|filter_nested_dict(["contributors"]) }}</p>
+                        <div class="card">
+                            <a class="card-link" href="{{ url_for('contribute.capability_details', contribution_id=capability['cont_id'], id=capability['id']) }}">
+                                <h3 class="capability_name">{{ capability|filter_nested_dict(["name"]) }}</h3>
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/templates/contribute/home.html:134]: The capability name font weight is bold according to Zeplin and not 800 px. Please fix this. You could have a separate CSS class for the specific changes for the capability/talent name. The rest could be inherited from h3.

File: contributions/catalog/webapps/static/css/style.css [hunk 2]
```diff
@@ -527,3 +551,70 @@ input.datadel-api {
 .hidetext { -webkit-text-security: disc; }
 .showtext { -webkit-text-security: none; }
 
+h1 {
+    font-size: 32px;
+    height: 36px;
+    font-weight: 800;
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/static/css/style.css:557]: Font weights for h1, h2, h3, etc., are `normal` according to the design. 800px is used only when the font in the design is ProximaNova-Extrabld. Please update this.

**Comment by sandeep-ps** [comment]
image
There is an overlap between the short description and the View Contribution button that needs fixing.

**Comment by minump** [comment]
converting to draft to work on feedback.

**Comment by minump** [comment]
The "View contribution" link is not visible for cards in the last row.
"View contribution" link is positioned at the bottom of the card. The main container is given padding of 50px and margin of 50px. 

Screen Shot 2023-01-31 at 10 25 40 AM

**Comment by minump** [comment]
> image
> 
> There is an overlap between the short description and the View Contribution button that needs fixing.

fixed by positioning "View contirubtion" link to the bottom of card.

File: contributions/catalog/webapps/static/css/style.css (hunk 2 above)
Comment (minump) [reply contributions/catalog/webapps/static/css/style.css:490]: Putting font-family:ProximaNova-Extrabld  -- doesnt show a difference. Can discuss this change.

**Comment by minump** [comment]
Some changes are made based on feedback. 
2 issues still persists : 
1. ProximaNova-Extrabld font-family doesnt seem to work.
2. The last row cards are not entirely visible as it is behind the footer.

Would like to get feedback on these.

**Comment by minump** [comment]
Making this PR "ready for review" with the caveat that 2 issues still needs to be resolved.

**Comment by minump** [comment]
> Some changes are made based on feedback. 2 issues still persists :
> 
> 1. ProximaNova-Extrabld font-family doesnt seem to work.
> 2. The last row cards are not entirely visible as it is behind the footer.
> 
> Would like to get feedback on these.

The issue on last row cards behind the footer - redirected to designer for solutions. Will await designer input.

**Comment by sandeep-ps** [comment]
>ProximaNova-Extrabld font-family doesnt seem to work

ProximaNova + font-weight: 800 is the same as ProximaNova-Extrabld, according to the Adobe Cloud account from which the font was downloaded.

**Comment by minump** [comment]
convert h4 font-weight to bold (change it from 500)

File: contributions/catalog/webapps/templates/contribute/home.html (hunk 1 above)
Comment (sandeep-ps) [reply contributions/catalog/webapps/templates/contribute/home.html]: Thanks.

**Comment by minump** [comment]
Removed container padding and margin as it will be worked on the footer PR.

Updated h4 font weight styling.

Ready for review

File: contributions/catalog/webapps/static/css/style.css (hunk 2 above)
Comment (sandeep-ps) [reply contributions/catalog/webapps/static/css/style.css:485]: Discussed about font changes and shared details about ProximaNova-Extrabld being ProximaNovv + font-weight: 800.

**Review by sandeep-ps — APPROVED** [review]
Approving. Thanks.

**Comment by minump** [comment]
Merging the PR

**Pull request merged**